### PR TITLE
feat: explore workspace

### PR DESCRIPTION
**Pull request by DefK1lla**

### CODE REVIEW

File: src/components/ExploreCubes/index.stories.tsx [hunk 1]
```diff
@@ -18,122 +18,398 @@ const Template: StoryFn<typeof ExploreCubes> = (args) => (
...
-      connectedComponent: 1,
-      title: "Tripdata",
-      name: "Tripdata",
-      type: "string",
-      operator: "string",
-      shortTitle: "string",
-      granularity: "string",
-      dimension: {
-        name: "string",
+    Airports: {
```
Comment (ifokeev) [inline src/components/ExploreCubes/index.stories.tsx:22]: too much data, should be saved somewhere like mocks

File: src/components/ExploreCubesSection/index.stories.tsx [hunk 2]
```diff
@@ -18,76 +18,162 @@ const Template: StoryFn<typeof ExploreCubesSection> = (args) => (
 export const Default = Template.bind({});
 
 Default.args = {
-  members: {
-    measures: [
+  selectedMembers: {
```
Comment (ifokeev) [inline src/components/ExploreCubesSection/index.stories.tsx:21]: too much data, should be saved somewhere like mocks

File: src/components/ExploreCubesSection/index.tsx [hunk 3]
```diff
@@ -319,19 +319,19 @@ const Cube = ({
     );
   };
 
-  return <>{SHOWN_CATEGORIES.map(getCategory)}</>;
+  return <div className={s.root}>{SHOWN_CATEGORIES.map(getCategory)}</div>;
```
Comment (ifokeev) [inline src/components/ExploreCubesSection/index.tsx:322]: why this needed? style changed?

File: src/components/ExploreDataSection/index.stories.tsx [hunk 4]
```diff
@@ -25,119 +25,5099 @@ const Template: StoryFn<typeof ExploreDataSection> = (args) => {
...
 
 Default.args = {
-  explorationRowId: "384904a6-e83c-47bf-b554-3faf8e55bc45",
-  height: 300,
-  isActive: true,
-  onExec: () => {},
-  queryState: {
-    columns: [
+  width: 825,
+  selectedQueryMembers: {
```
Comment (ifokeev) [inline src/components/ExploreDataSection/index.stories.tsx:29]: too much data, should be saved somewhere like mocks

File: src/components/ExploreFiltersSection/index.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,422 @@
...
+const Template: StoryFn<typeof ExploreFiltersSection> = (args) => (
+  <RootLayout>
+    <ExploreFiltersSection {...args} />
+  </RootLayout>
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  availableQueryMembers: {
```
Comment (ifokeev) [inline src/components/ExploreFiltersSection/index.stories.tsx:21]: too much data, should be saved somewhere like mocks

File: src/components/ExploreWorkspace/index.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,403 @@
...
+  </RootLayout>
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  basePath:
+    "/~/explore/35c549a8-c38a-4ff1-90a5-b3081a35aa93/b24d941c-94ac-4bca-8a3a-9e179a4fc06e",
+  loading: false,
+  meta: [
```
Comment (ifokeev) [inline src/components/ExploreWorkspace/index.stories.tsx:25]: too much data, should be saved somewhere like mocks

File: src/components/ExploreWorkspace/index.tsx [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+import useDimensions from "@/hooks/useDimensions";
+import ExploreFiltersSection from "@/components/ExploreFiltersSection";
+import AppLayout from "@/layouts/AppLayout";
+
+import type { FC, ReactNode } from "react";
+
+const DEFAULT_ROW_HEIGHT = 20;
+
+interface ExploreWorkSpaceProps {
+  basePath: string;
```
Comment (ifokeev) [inline src/components/ExploreWorkspace/index.tsx:17]: not used anywhere

File: src/components/ExploreCubesSection/index.tsx (hunk 3 above)
Comment (DefK1lla) [reply src/components/ExploreCubesSection/index.tsx:322]: yes, background

File: src/components/ExploreDataSection/index.stories.tsx [hunk 8]
```diff
@@ -10,134 +12,46 @@ export default {
...
+    setColumns(
+      args.queryState.columns.map((c) => {
+        const header = c["Header" as keyof typeof c] as string;
+        const newHeader = settings.hideCubeNames
+          ? header.split(" ")[1]
+          : header;
+        return { ...c, Header: newHeader };
+      })
+    );
+  }, [args.queryState.columns, args.queryState.rows, settings]);
```
Comment (ifokeev) [inline src/components/ExploreDataSection/index.stories.tsx:40]: code duplication

File: src/components/ExploreFiltersSection/index.stories.tsx [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+        const newState = { ...prev };
+        const memberIndex = prev.filters.findIndex(
+          (m) => JSON.stringify(m) === JSON.stringify(member)
+        );
+        newState.filters[memberIndex] = newMember;
+        return newState;
+      }),
+    remove: (_: CubeMember) => setMembers({}),
+    add: () => {},
+  });
```
Comment (ifokeev) [inline src/components/ExploreFiltersSection/index.stories.tsx:33]: code duplication

**Review by ifokeev — CHANGES_REQUESTED** [review]

**Review by ifokeev — APPROVED** [review]

**Pull request merged**